code/index.html

```diff
@@ -4,34 +4,108 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./normalize.css" />
     <title>Technigo Project Cluedo</title>
   </head>
   <body>
+    <!-- BACKGROUND -->
+    <div class="background">
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <!-- SOURCE for Background idea, which was modified from the original: https://dev.to/vincenius/websites-for-creative-backgrounds-1mao -->
+    <div class="title">
+      <h1>CLUEDO</h1>
+    </div>
+    <div class="title-p">
+      <p>
+        Choose your suspect, the room and a weapon to reveal the crime.
+        <br />You may shuffle the cards as many times as you'd like to, but the
+        mystery won't be revealed unless all three decks are shuffled! When
+        you're finished, click on the 'New Round' button.
+      </p>
+    </div>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p>SUSPECT</p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
+        <img id="killerImage" src="" height="150" />
         <p id="killerDescription"></p>
+        <p id="favouriteWeapon"></p>
+      </article>
```

Comment:
I saw that you wrote that you did not use an extension for indentation for this project. However, I think that you indentation looks great, and the code is structured and clean. Great job. 🥳

Reply:
Agree with this! It's great to practice formatting by hand 👍